package.json

```diff
@@ -456,6 +477,11 @@
       "type": "object",
       "title": "PowerShell Configuration",
       "properties": {
+        "powershell.sideBar.CommandExplorerVisibility": {
```

Comment:
It looks like the `sideBar` is actually called the "Activity Bar". Can you change this to `activityBar`?

Reply:
```suggestion
        "powershell.activityBar.CommandExplorerVisibility": {
```